src/NUnitEngine/nunit.engine/Runners/MasterTestRunner.cs

```diff
@@ -351,6 +351,9 @@ private TestEngineResult RunTests(ITestEventListener listener, TestFilter filter
             var eventDispatcher = new TestEventDispatcher();
             if (listener != null)
                 eventDispatcher.Listeners.Add(listener);
+
+            FindExtensionsInTestPackage(this.TestPackage);
+
```

Comment:
I'm not sure we want to do this, but if we did I believe we would want the engine to do it by itself, without requiring the console to do quite so much work. If we make it a console option, then we would just pass a new setting in the test package itself.